panel/src/fiber/app.js

```diff
@@ -14,24 +14,32 @@ export default {
     Fiber.init({
       page: window.fiber,
       component: (name) => Vue.component(name),
-      props: (props) => {
+      props: (data) => {
+
+        /** Set globals */
+        ["$config", "$system", "$translation", "$urls"].forEach(key => {
```

Comment:
Why are we setting these differently than the others?
I don't think you'll need the else-statement here. Our `protocol.js` keeps the globals from the initial request around and adds them after every partial request anyways. So this method sees always a full `data` object.

Reply:
I didn't get it running without this if clause.